### ISSUE TITLE
Mystery Text discussion of Radio Plays (any plays other than ac.txt)

### ISSUE THREAD

**Issue by ebeshero** [newtfire/introDH-Hub#46]
Post your ideas and findings and screen captures about any of the Radio Play texts here.

**Comment by gak5275** [comment]
![bellscreenshot_voyant](https://user-images.githubusercontent.com/89995510/136712591-dfdc5c37-f9c8-40ec-9517-b3d910644561.PNG)
![bellscreenshot_antconc1](https://user-images.githubusercontent.com/89995510/136712596-4954f696-6c7f-447b-9354-cd9930cc255c.png)
![bellscreenshot_antconc2](https://user-images.githubusercontent.com/89995510/136712601-ceb9898e-fa15-4925-878a-cd81867df6ec.png)

I looked at **belltelephone.txt** in the radio plays. This appears to be some sort of reenactment of when Alexander Graham Bell first showed off the telephone that he invented. At first I thought that it was the real thing, but the presence of a narrator indicates that this is just acting. Apart from Bell himself, there appear to be two other important named characters, Watson and Hubbard. I looked up those names in relation to Alexander Graham Bell and found out that Watson is Thomas Augustus Watson, who was Bell's assistant. Hubbard is most likely either Mabel Gardiner Hubbard, who was a businesswoman and Bell's wife, or her father Gardiner Greene Hubbard, who was a lawyer.

**Comment by AlexanderVaughan** [comment]
![VoyabtCapture](https://user-images.githubusercontent.com/89950828/137005153-b30263a3-a8ca-4092-bce4-582f7d640036.PNG)
![Ant1Capture](https://user-images.githubusercontent.com/89950828/137005170-59199a63-14c7-4283-addb-fbdc3ad76de6.PNG)
![NT2Capture](https://user-images.githubusercontent.com/89950828/137005189-b48aee3c-1852-4bfd-8ba0-d4775e785c35.PNG)

I looked at anesthesia.txt in the radio plays. Based on the analysis it seems to be a story about anesthetic gas. There seem to be a few characters including Horace Wells, a man named Coltan and another named Clooney. It seems as thought there is some sort of show of entertainments based around this gas as well. I looked into the story a bit and found Horace Well was a dentist who was looking for a way to ease the pain of his patients.

**Comment by Yuying-Jin** [comment]
I choose ants.txt. According to the name of the file, it seems to introduce ants, so I search the keyword "like" to learn what ants like to do or to eat. However, this txt file shows how Leiningen prevents ants from his plantation. He stops the ants from entering the plantation through the water canal. Then he kills the ants by flooding or flames, and finally he kills all of them.
![Jin_10_12_AntConc_ex2_1](https://user-images.githubusercontent.com/62861214/137007571-a7674041-f82c-47f1-a6dd-2711d6f11625.PNG)
![Jin_10_12_AntConc_ex2_2](https://user-images.githubusercontent.com/62861214/137007572-1fab3c45-1497-4e71-8088-3c5355aba8f6.PNG)
![Jin_10_12_AntConc_ex2_3](https://user-images.githubusercontent.com/62861214/137007573-34773ad1-6462-4827-898a-9bafc4220caf.PNG)

**Comment by AloraBenitez** [comment]
![Screenshot (53)](https://user-images.githubusercontent.com/89956616/137015691-beab643e-3488-4700-b6cf-d4d64bd2936e.png)

![Screenshot (54)](https://user-images.githubusercontent.com/89956616/137015710-a980fd95-19d3-4987-aeaf-3d1eb20d1578.png)

![Screenshot (55)](https://user-images.githubusercontent.com/89956616/137015723-4c087e9a-aaa1-474e-96d7-84cf4c538ad9.png)

I went into Radio play and chose to look into the cu.text. Based on the layout I found it could have been a play based on an independent producer who signs two actors. Later on it describes the events that occur on set.  Aside from the story itself, it was interesting that voyant tools picked up on the amount of times it mentions "her" 65 times however ant conc noticed " I'm" 22 times. It was pretty interesting to find this and made me question weather ant conc is aware that the term "her" isn't necessarily apart of the story and if it recognizes that this only describes who is speaking.
